src/main/kotlin/community/flock/eco/workday/services/email/CostExpenseMailService.kt

```diff
@@ -2,39 +2,42 @@ package community.flock.eco.workday.services.email
 
 import community.flock.eco.workday.config.properties.MailjetTemplateProperties
 import community.flock.eco.workday.model.CostExpense
+import community.flock.eco.workday.utils.DateUtils.toHumanReadable
 import org.slf4j.Logger
 import org.slf4j.LoggerFactory
 import org.springframework.stereotype.Service
 
 @Service
-class CostExpenseMailService(private val emailService: EmailService, private val mailjetTemplateProperties: MailjetTemplateProperties) {
+class CostExpenseMailService(
+    private val emailService: EmailService,
+    private val mailjetTemplateProperties: MailjetTemplateProperties,
+) {
     private val log: Logger = LoggerFactory.getLogger(CostExpenseMailService::class.java)
 
-    fun sendUpdate(
-        old: CostExpense,
-        new: CostExpense,
-    ) {
-        val recipient = new.person
-
-        var subject = "Update in CostExpense!"
-        var emailMessage = "Er is een update in CostExpense."
-
-        if (old.status !== new.status) {
-            subject = "Status update in CostExpense!"
-            emailMessage = "De status van je CostExpense is veranderd.\n\n" +
-                "vorige status: ${old.status}.\n" +
-                "nieuwe status: ${new.status}."
-        }
-
+    fun sendUpdate(expense: CostExpense) {
+        val recipient = expense.person
+
+        val subject = "Declaratie update - ${expense.description ?: "beschrijving onbekend"}!"
+        val emailMessage =
+            """
+            <p>Je declaratie voor '${expense.description ?: "onbekend"}' is bijgewerkt.<p>
+
+            <ul>
+                <li>Beschrijving: ${expense.description}</li>
+                <li>Kosten gemaakt op: ${expense.date.toHumanReadable()}</li>
+                <li>Bedrag: ${expense.amount}</li>
+                <li>Status: ${expense.status}</li>
```

Comment:
Willen we die status nog een leesbare (vertaalde?) variant geven? Zelfde vraag voor bijv de types van de Expense of LeaveDay

Reply:
Niet voor nu. Imo kunnen mensen dit nu prima vertalen. Zeker als we de maitjes toch netjes in het Engels gaan doen (alles in die app is Engels)